core/prometheus_exporter_test.go

```diff
@@ -0,0 +1,11 @@
+package core
+
+import "testing"
+
+func TestCollector(t *testing.T) {
+	// TODO
```

Comment:
I've added a basic test just to validate that the metrics were being collected from the context.

Ideally I wanted to add an end to end test, however, I was having some issues with IPVS when creating services.

Not sure if you see any issues with trying to spin an instance of gorb and test the HTTP APIs from an end to end perspective. If we had an instance of gorb running it would be easy to spin up prometheus and actually check if the metrics are being collected.

Anyway, let me know your thoughts. I have some free cycles so I wouldn't mind tackling the test suite for gorb.